release_bot/github.py

```diff
@@ -395,6 +402,24 @@ def close_issue(self, number):
         self.logger.error(f'Failed to close issue #{number}')
         return False
 
+    def put_labels_on_issue(self, number, labels):
+        """
+        Put labels on Github issue or PR
+        :param number: number of issue/PR
+        :param labels: list of str
+        :return: True on success, False on fail
+        """
+        payload = {'labels': labels}
+        url = (f"{self.API3_ENDPOINT}repos/{self.conf.repository_owner}/"
+               f"{self.conf.repository_name}/issues/{number}")
+        self.logger.debug(f'Attempting to put labels on issue/PR #{number}')
+        response = requests.patch(url=url, headers=self.headers, json=payload)
+        if response.status_code == 200:
+            self.logger.debug(f'Labels put on issue #{number}')
+            return True
+        self.logger.error(f'Failed to put labels on issue #{number}')
```

Comment:
I think it would be good to include what those labels were, to make debugging easier.

Reply:
+1 , I will change it.